### PR TITLE
add csharpls

### PR DESCRIPTION
**Pull request by chen244**
add csharpls, it works for me

### CODE REVIEW

**Comment by chen244** [comment]
image
It works well!

**Comment by chen244** [comment]
o , I got a mistake.... please use the last pr, not the add diffs...

**Review by williamboman — APPROVED** [review]
Thanks 🌟 !

File: lua/nvim-lsp-installer/_generated/metadata.lua [hunk 1]
```diff
@@ -211,4 +211,4 @@ return {
   zls = {
     filetypes = { "zig", "zir" }
   }
-}
+}
```
Comment (williamboman) [inline lua/nvim-lsp-installer/_generated/metadata.lua]: Btw @chen244 these files under the `_generated/` directory are automatically generated by running the `./scripts/autogen_metadata.sh` script :)

**Pull request merged**